File: help/eng_gb/exercicios_da_ficha_5/Ficha05_010/Ficha05_010.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha05_010 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        int minimum = 999 ;
        for( int i = 1; i < 6; i = i + 1 ) {
            System.out.print(i + "º grade : " );
            int grade = Integer.valueOf(keyb.nextLine());
            if(grade < minimum ){
                minimum = grade ;
            }
        }
        System.out.print("Nota mínima = " + minimum );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Sep 24 22:05

File: help/eng_gb/exercicios_da_ficha_5/Ficha05_010/Ficha05_010.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    var minimum = 999 ;
    for( var i = 1; i < 6; i = i + 1 ) {
        document.write(i + "º grade : " );// print text
        grade = parseInt(prompt(i + "º grade : " ,"0") );
        
        document.write(grade + "\n");// print input 
        if(grade < minimum ){
            minimum = grade ;
        }
    }
    document.write("Nota mínima = " + minimum );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Sep 24 22:05
